pyproject.toml

```diff
@@ -1,20 +1,31 @@
-[build-system]
-requires = ["flit_core >=2,<4"]
-build-backend = "flit_core.buildapi"
+[tool.poetry]
+name = "redact"
+version = "5.1.0"
+description = "Command-line and Python client for Brighter AI's Redact"
+authors = ["brighter AI <dev@brighter.ai>"]
+readme = "README.md"
+license = "MIT License"
+
+[tool.poetry.scripts]
+redact_file = "redact.main:redact_file_entry_point"
+redact_folder = "redact.main:redact_folder_entry_point"
 
-[tool.flit.scripts]
-redact_file = "redact.__main__:redact_file_entry_point"
-redact_folder = "redact.__main__:redact_folder_entry_point"
+[tool.poetry.dependencies]
+python = "^3.7"
+httpx = "^0.22.0"
```

Comment:
Downgrade from 0.23.0?

Reply:
No really. It's defined that this application needs at least httpx = 0.22.0. All versions that have the same major version are fine. Poetry then runs a dependency resolution process looking for the highest version of each package that is still compatible with the rest. The final currently installed version is pinned inside the poetry.lock file. The lock file makes sure that you won't have any surprises since every package has an exact version pinned down.

Reply:
The question is if we really need httpx 0.23.0 or not for this tool to function correctly. The more restrictive we make the version numbers to harder it gets to make it work alongside other packages due to potential version conflicts.

Reply:
Ok, if that is including 0.23 . 0.23 includes fixes for vulnerabilities, so it should be the default.